### PR TITLE
Fix the documentation building.

### PR DESCRIPTION
**Pull request by mithro**
* Add the missing `.readthedocs.yml` file.
 * Use hdl organization's hdlparse fork.
 * Add missing `third_party/make-env` submodule.
 * Revert the `docs/Makefile` to match skywater-pdk's version.

### CODE REVIEW

**Comment by mithro** [comment]
Seems to be working now - https://sky130-fd-pr-reram--11.org.readthedocs.build/en/11/user_guide.html

File: docs/Makefile [hunk 1]
```diff
@@ -53,13 +61,23 @@ livehtml: | $(CONDA_ENV_PYTHON)
 		--ignore _build \
         	--ignore .git \
         	--ignore */4913 \
-		--host $(HOSTNAME)\
-		--port 8080 \
```
Comment (QuantamHD) [inline docs/Makefile:57]: We want to keep this part so that when it binds to a host you can specify a DNS name.

**Comment by QuantamHD** [comment]
Will merge this in

**Comment by QuantamHD** [comment]
This is in the repo. 👍

**Pull request closed without merging**